src/App.jsx

```diff
@@ -153,9 +153,68 @@ const Home = () => (
                     </li>
                 </ul>
             </div>
-        </div>        <div className="row">
+        </div>
+        <div className="row">
             <Preview />
         </div>
+        <div className="row">
+            <div className="card col-md-4">
+                <div className="card-body">
+                    <h2 className="card-title">
+                       About
+                    </h2>
+                    <p className="card-text">Stats Interactives are a set of interactives used to illustrate introductory concepts in statistics. The project was built by Prof. Doru Cojoc in collaboration with the Center for Teaching and Learning.</p>
```

Comment:
feels more like CTL did the building...is there another way to phrase this?

Reply:
Yeah, I agree this is awkward.  This is how its phrased for Econ Practice, so I just cribbed that. I'll come up with something different.